tests/foreman/cli/test_fact.py

```diff
@@ -25,22 +19,14 @@ def test_list_success(self, fact):
         @Assert: Fact List is displayed
 
         """
-
-        args = {
-            'search': "fact='%s'" % fact,
-        }
-
-        facts = Fact().list(args)
-
-        self.assertEqual(facts[0]['fact'], fact)
-
-    @data(
-        gen_string('alpha', 10),
-        gen_string('alpha', 10),
-        gen_string('alpha', 10),
-        gen_string('alpha', 10),
-    )
-    def test_list_fail(self, fact):
+        for fact in ('uptime', 'uptime_days', 'uptime_seconds', 'memoryfree',
+                     'ipaddress'):
+            with self.subTest(fact):
+                args = {'search': "fact='%s'" % fact}
```

Comment:
Yep created an issue to tackle all of these in the entire project: https://github.com/SatelliteQE/robottelo/issues/2853